sigsci_site_manager/deploy.py

```diff
@@ -112,12 +111,23 @@ def create_integrations(api, data):
             print('    Failed: %s' % e)
 
 
-def generate_advanced_rules_request(api, source, data):
-    print('\n\nEmail support@signalsciences.com with the following...\n')
-    print('Please copy the following advanced rules from %s/%s to %s/%s:' %
-          (source['corp'], source['site'], api.corp, api.site))
+def create_advanced_rules(api, source, data):
+    print('Creating advanced rules...')
     for item in data:
-        print('    %s (ID %s)' % (item['shortName'], item['id']))
+        if '+' in item['shortName']:
+            print("    Warning for %s: Rule names containing '+' not supported."
+                  % item['shortName'])
```

Comment:
If there are advanced rules with '+' in the name it should probably still print out the email template for those specific rules so they can be copied. It should also `continue` on the loop so that unsupported rule names don't get a copy attempt that will fail, right?

Reply:
Changed it to just catch any failed copy and output an email request at the end.